### ISSUE TITLE
Look at Helium 

### ISSUE THREAD

**Issue by jd-campbell** [soybase/jekyll-soybase#123]
https://github.com/cardinalb/helium-docs/wiki

**Comment by maxglycine** [comment]
I don't think we are still looking at Helium, are we?
@StevenCannon-USDA @jd-campbell

**Comment by StevenCannon-USDA** [comment]
> I don't think we are still looking at Helium, are we? @StevenCannon-USDA @jd-campbell

It's been back-burnered, but I don't think we've abandoned it. We still need a pedigree viewer. Helium is nice. One of the developers (Sebastian Raubach) gave some instructions in May about how to send data via URL into their hosted instance:

```
Basically, on the Germinate side, we generate a .zip file containing both the pedigree and, optionally, trait data.
The pedigree file *has* to be called “pedigree.helium” and the trait file *has* to be called “pedigree-attributes.helium”.
They both obviously have to follow the supported file formats of Helium.
The generated .zip file has to be accessible publicly on a web server without authentication.
Germinate then generates a link the user can click on, that’s of the form: [https://helium.hutton.ac.uk/#/pedigree?germinateUrl=https%3A%2F%2Fics.hutton.ac.uk%2Fgerminate-demo%2Fapi%2Fdataset%2Fexport%2Fasync%2F04af1e2d-b931-4f60-8571-6b54d216e4e8%2Fdownload](https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fhelium.hutton.ac.uk%2F%23%2Fpedigree%3FgerminateUrl%3Dhttps%253A%252F%252Fics.hutton.ac.uk%252Fgerminate-demo%252Fapi%252Fdataset%252Fexport%252Fasync%252F04af1e2d-b931-4f60-8571-6b54d216e4e8%252Fdownload&data=05%7C02%7Csteven.cannon%40usda.gov%7Cf0109f232b4d4fec4cf108dc765cc6d7%7Ced5b36e701ee4ebc867ee03cfa0d4697%7C1%7C0%7C638515388077567513%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=lpav9n768H17f3Uaq86by3SzYLw659sv6%2BWRHSwUDwM%3D&reserved=0)
Basically: https://helium.hutton.ac.uk/#/pedigree?germinateUrl={URL TO YOUR ZIP BUT URL-ENCODED}
This will open Helium on the import screen for Germinate data with your URL loaded.
You can then click import to load the data.
```
